Jenkinsfile

```diff
@@ -61,6 +62,40 @@ pipeline {
                 }
             }
         }
+        stage('Staging') {
+            when {
+                 not {
+                     branch 'main'
```

Comment:
This includes "javadoc" and "asf-site" branches - is that OK?

Reply:
It only runs for branches that have a `Jenkinsfile` so those branches will never get built by Jenkins

Reply:
Good!